etherboy-inventory.yaml

```diff
@@ -9,37 +9,26 @@ all:
     aws_ami_id: ami-a4dc46db
     aws_instance_type: t2.medium
     aws_region: us-east-1
-    aws_vpc_id:
-    aws_vpc_subnet:
-    aws_instance_profile:
-    aws_keypair_name:
+    aws_vpc_id: vpc-5cfef627
```

Comment:
This repository is meant to serve as a guide to accompany the developer docs. I think we shouldn't put in specifics here. Maybe we need to move these elsewhere. What do you think @ksandom ?

Reply:
I agree.
One way this could be achieved is to have a `vars` or `global_vars` folder where settings like this can be placed. Then we simply include our specifics in a file that we place there, and other people can do the same for their infrastructure.

https://docs.ansible.com/ansible/latest/modules/include_vars_module.html
https://docs.ansible.com/ansible/latest/user_guide/playbooks_best_practices.html#group-and-host-variables